### PR TITLE
New version: RoyalApps.RoyalTS.V7 version 7.2.50703.0

### LINKED ISSUE #161478: [Update Request]: RoyalTS V7 7.02.50703
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

RoyalApps.RoyalTS.V7

### Package Version

7.02.50703

### Please describe the changes you would like to see

_No response_

### PR DESCRIPTION
**Pull request by StefanKoell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161284)

Resolve https://github.com/microsoft/winget-pkgs/issues/161478

### CODE REVIEW

**Comment by Dragon1573** [comment]
Hi @StefanKoell, 👋🏼

Could you please add `Resolve #161478` to your [very first comment](https://github.com/microsoft/winget-pkgs/pull/161284#issue-2388079752) of this package? When this PR is merged, the related issue can be automatically closed as completed.

This can also prevent other contributors from creating duplicate PRs. ❤️

**Comment by StefanKoell** [comment]
@Dragon1573 forgive my ignorance. I'm not sure I can follow. What exactly should I do?

**Comment by Dragon1573** [comment]
Just simply add following contents to your first comment in this PR.

> Resolve #161478

**Comment by StefanKoell** [comment]
Done. Hope it is correct.

File: manifests/r/RoyalApps/RoyalTS/V7/7.2.50703.0/RoyalApps.RoyalTS.V7.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+PackageIdentifier: RoyalApps.RoyalTS.V7
+PackageVersion: 7.2.50703.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+ProductCode: '{4959A733-614C-44E8-AC0D-3E7BCA9365AC}'
+AppsAndFeaturesEntries:
+- DisplayVersion: 7.2.50703.0
+  UpgradeCode: '{4959A733-614C-44E8-AC0D-3E7B5A7375AB}'
```
Comment (Trenly) [inline manifests/r/RoyalApps/RoyalTS/V7/7.2.50703.0/RoyalApps.RoyalTS.V7.installer.yaml:12]:
```suggestion
- UpgradeCode: '{4959A733-614C-44E8-AC0D-3E7B5A7375AB}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**